improver_tests/utilities/test_spatial.py

```diff
@@ -433,23 +436,29 @@ class Test_transform_grid_to_lat_lon(IrisTest):
 
     def setUp(self):
         """Set up the cube."""
-        self.cube = set_up_cube(zero_point_indices=((0, 0, 1, 1),), num_grid_points=2)
-        self.cube.coord(axis="x").points = np.array([-1158000.0, 924000.0])
-        self.cube.coord(axis="y").points = np.array([-1036000.0, 902000.0])
+        data = np.ones((1, 2, 2), dtype=np.float32)
+        self.cube = set_up_variable_cube(
+            data,
+            "precipitation_amount",
+            "kg m^-2",
+            "equalarea",
+            grid_spacing=2000000,
+            domain_corner=(-1000000, -1000000),
+        )
 
     def test_transform_grid(self):
         """
         Test transformation of grid
         """
         expected_lons = np.array(
-            [-17.11712928, 9.21255933, -24.5099247, 15.27976922]
+            [-15.23434831, 10.23434831, -22.21952954, 17.21952954]
         ).reshape(2, 2)
         expected_lats = np.array(
-            [44.51715281, 44.899873, 61.31885886, 61.9206868]
+            [45.10433161, 45.10433161, 62.57973333, 62.57973333]
         ).reshape(2, 2)
         plugin = transform_grid_to_lat_lon
         result_lats, result_lons = plugin(self.cube)
         self.assertIsInstance(result_lats, np.ndarray)
         self.assertIsInstance(result_lons, np.ndarray)
-        self.assertArrayAlmostEqual(result_lons, expected_lons)
-        self.assertArrayAlmostEqual(result_lats, expected_lats)
+        assert_almost_equal(result_lons, expected_lons)
+        assert_almost_equal(result_lats, expected_lats)
```

Comment:
Any particular reason for using `numpy.testing` here?

Reply:
Because I was going to trim the number of decimal places being checked, and then I decided not to. I also prefer to use numpy tests rather than iris; we will one day be free of IrisTest, one line at a time.